Paintroid/src/main/java/org/catrobat/paintroid/ui/viewholder/TopBarViewHolder.java

```diff
@@ -90,4 +90,18 @@ public int getHeight() {
 	public void removeItem(int item) {
 		moreOptions.removeItem(item);
 	}
+
+	@Override
+	public void removeStandaloneMenuItems(Menu menu) {
+		removeItem(R.id.pocketpaint_nav_save_image);
```

Comment:
you are calling `removeItem()`. You should call `menu.removeItem()`. The idea is to not have the menu as a member variable.